05-onchain.md

```diff
@@ -513,7 +513,15 @@ A local node:
   using the revocation private key.
   - SHOULD extract the payment preimage from the transaction input witness, if
   it's not already known.
-  - MAY use a single transaction to *resolve* all the outputs.
+  - if `option_anchor_outputs` applies:
+    - MAY use a single transaction to *resolve* all the outputs.
+    - if confirmation doesn't happen before reaching `security_delay` blocks from
+  expiry:
+      - MUST *resolve* revoked outputs in their own, separate penalty transactions. A previous
```

Comment:
Feels like `SHOULD` is more correct here, since the sweeping node is aware of the risk and should be allowed to do as it pleases.

Reply:
I agree, @ariard I think this is the last pending point before we merge, what do you think?

Reply:
We've got the ACKs on this, should we wait til this is addressed? paging @ariard

Reply:
Sounds good to me, it's up to the implementer to gauge risk-vs-fee-saving.